sql/parser/walk.go

```diff
@@ -86,6 +86,11 @@ func WalkExpr(v Visitor, expr Expr) Expr {
 	case *QualifiedName:
 		// Terminal node: nothing to do.
 
+	case Row:
```

Comment:
`TestFillArgs` tried to be exhaustive of the expression nodes. It failed when a new one was added.

In the case of `Row`, I think it makes sense to add a `NormalizeExpr` walk that transforms `Row` into `Tuple` before `FillArgs` and `EvalExpr` are called. `Row` and `Tuple` are synonymous for execution and only differ in the SQL they generate. There are a variety of other normalizations we'll want to perform as well, such as pre-evaluating branches of the expression tree which are constant.